nautobot/ipam/graphql/types.py

```diff
@@ -12,6 +12,7 @@ class IPAddressType(gql_optimizer.OptimizedDjangoObjectType):
     family = graphene.Int()
     interface = graphene.Field("nautobot.dcim.graphql.types.InterfaceType")
     vminterface = graphene.Field("nautobot.virtualization.graphql.types.VMInterfaceType")
+    # TODO(timizuo): What would i do to this
     nat_outside = graphene.Field(lambda: IPAddressType)
```

Comment:
Should this be changed to nat_outside_list

Reply:
Probably should be a `graphene.List` of `IPAddressType` yeah - needs to be tested though.

Reply:
How do i test this

Reply:
Might "just work" actually - try adding some appropriate data, then go to `/graphql/` and run a query like:

```graphql
query {
  ip_addresses (limit: 10) {
    address
    nat_outside_list {
      address
    }
  }
}
```

On demo.nautobot.com this seems to be working already:

![image](https://user-images.githubusercontent.com/5603551/229812534-67d32759-c02f-4df2-a543-64e4a200cf86.png)